src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -513,8 +514,26 @@ public record Pool(
 ) : StatefulEntityBase<PoolState>(State);
 
 
-// TODO
-public record AgentConfig();
+public record ClientCredentials
+(
+    Guid ClientId,
+    string ClientSecret
```

Comment:
How are we protecting this value? Or, maybe a better question, are we making sure this doesn't get logged anywhere?

Reply:
same behaviour as in python code